tests/acceptance/features/webUIUpload/upload.feature

```diff
@@ -16,6 +16,18 @@ Feature: File Upload
     And file "new-lorem.txt" should be listed on the webUI
     And as "user1" the content of "new-lorem.txt" should be the same as the local "new-lorem.txt"
 
+  @smokeTest
+  Scenario: simple upload of a folder that does not exist before
+    When the user uploads a folder containing following files using the webUI:
```

Comment:
```suggestion
    When the user uploads a folder containing the following files using the webUI:
```

Reply:
```suggestion
    When the user uploads a folder containing the following files in separate sub-folders using the webUI:
```
Maybe if we mention about the sub-folders in this step, then the step about `the last uploaded folder should contain the following files inside the sub-folders` will not be so surprising.

And at least we give the reader some clue that we are doing a "special" sort of folder-structure-upload.